bazel/repositories.bzl

```diff
@@ -82,7 +82,18 @@ def proxy_wasm_cpp_host_repositories():
         patch_args = ["-p1"],
     )
 
-    # Core.
+    # Core deps. Try to keep up with Envoy versions:
+    # https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl
```

Comment:
Why does it matter whether C++ Host version match Envoy's? The builds in Envoy don't rely on versions in here, and the builds in here don't rely on anything in Envoy.

A lot of dependencies were not updates in the past 2 years, so things broke, but virtually all of those issues are due to the changes in compilers and build rules/systems, and have nothing to do with Envoy.

Reply:
Errr it doesn't have to match exactly, but we also can't lag dependencies by years (the current state), given that part of the CppHost verification process is to patch a draft PR into Envoy and ensure everything works. Version mismatches are a drag on productivity in this repo. I don't know of a systematic way to address this since Envoy likes to be in control of its dependencies.

Reply:
> Errr it doesn't have to match exactly, but we also can't lag dependencies by years (the current state)

Right, but that can be shortened to "Keep them updated." ;)

> given that part of the CppHost verification process is to patch a draft PR into Envoy and ensure everything works.

That should require only bumping C++ Host, and sporadically a Wasm engine dependency, if that's what's updated behind the scenes.

There is a separate issue with Wasmtime, but that's because we've embedded Wasmtime's Bazel rules into this repository. I've suggested splitting it into `wasmtime-bazel` multiple times in the past, since it really shouldn't be here.

Reply:
> shortened to "Keep them updated." ;)

Fair, done.